### DISCUSSION TITLE
Διχοτόμηση της καρτέλας προσωπικό UI Design

### DISCUSSION THREAD

**Discussion by Alkissourvinos** [ioniodi/sitegr#244] [Q&A]
### Δήλωση Θέματος
- Θέλω να διασπάσω την καρτέλα προσωπικό ανάλογα με τις βαθμίδες των καθηγητών και του διοικητικού προσωπικού, Δηλαδή να σπάει σε δύο καρτέλες, τους καθηγητές και το διοικητικό προσωπικό. Μετά από αυτό θέλω να ξεχωρίσω τους καθηγητές σε επισκέπτες καθηγητές, Τεχνικό προσωπικό/ εργαστηριακοί και Ακαδημαϊκοί υπότροφοι. Δηλαδή θέλω να κάνω ένα drop-down menu στο tab προσωπικό στο οποίο να είναι οι παραπάνω κατηγορίες και ταυτόχρονα θα έχει την δυνατότητα κανείς να δει ολόκληρο το προσωπικό όπως και τωρα
- Εκτιμώμενη δυσκολία:  hard
- Σχετικοί συνεργατές: @
- Σχετικά θέματα: Αλλαγή UI design

### Υπενθυμίσεις
- [x] Έχω βάλει σύντομο τίτλο (π.χ., το όνομα της σελίδας ή της ανακοίνωσης) και έχω ενημερώσει τα παραπάνω πεδία με περισσότερες λεπτομέρειες (π.χ., λινκ σε αρχεία ή φακέλους που αλλάζουν ή εξωτερικές πηγές) για τις προσθήκες που θα κάνω
- [x] Έχω ελέγξει ότι **το περιέχομενο των αλλαγών μου δεν το έχει δηλώσει κάποιος άλλος** σε προηγούμενο issue ανοιχτό ή κλειστό
- [x] Πριν προχωρήσω σε pull request θα πάρω ετικέτα green light
- [x] Για την δεύτερη συνεισφορά έχω υλοποιήσει μια νέα λειτουργία και έχω εντοπίσει δύο σχετικούς αξιολογητές όπως φαίνεται στο λινκ που δίνω παραπάνω, διαφορετικά δεν γίνονται δεκτά θέματα που αφορούν απλή συνεισφορά περιεχομένου όπως στην πρώτη συνεισφορά

**Comment by epidrome** [comment]
αρχικά, ευχαριστούμε για το ενδιαφέρον και τον ενθουσιασμό σου!

θέτεις αρκετά θέματα τα οποία καλύτερα να γίνουν συζήτηση πριν γίνουν ένα ή περισσότερα θέματα που θα κλείσουν με τα αντίστοιχα αιτήματα ενσωμάτωσης.

για να γίνει καλύτερη συζήτηση θα πρέπει να ενημερώσεις την πρόταση σου με συνδέσμους προς σχετική τεκμηρίωση και παραδείγματα.
